### ISSUE TITLE
Could this be used with real time data?

### ISSUE THREAD

**Issue by dejongyeong** [heethesh/lidar_camera_calibration#5]
Hi, I would like to know does this package work if I doesn't use rosbag? I'll launch camera and velodyne in different terminal that publish the require topic as mentioned in the README.md.

Thanks.

**Comment by heethesh** [comment]
Yes it should work, you can still launch the GUI and pick (and save) corresponding 2D/3D points.

**Comment by dejongyeong** [comment]
Hi @heethesh, thanks for getting back to me, do I need to save the calibrated YAML file?

**Comment by heethesh** [comment]
Yes, if you save it in `/home/.ros` folder (not sure) - which is the default save path, I believe ROS will automatically pick the data from here for a given camera name.

**Comment by dejongyeong** [comment]
Ok @heethesh Thanks for the information. I'll give it a try and see how it goes. I might need some pointers if I have some difficulties. Thanks.

**Comment by heethesh** [comment]
No problem, you may want to update the point-cloud visible ranges for the GUI [here](https://github.com/heethesh/lidar_camera_calibration/blob/master/scripts/calibrate_camera_lidar.py#L232). Working on a new update that uses Rviz GUI instead of Matplotlib which should be more convenient to work with.

**Comment by dejongyeong** [comment]
Ok. I'm new to ROS and will look into details once I got that to work. Thanks.

**Comment by dejongyeong** [comment]
Hi @heethesh, one quick question that I would like to ask, does it work for LiDAR 32 channel?

**Comment by heethesh** [comment]
yes it should work
